algorithmic_efficiency/workloads/cifar/cifar_jax/workload.py

```diff
@@ -19,6 +21,29 @@
 
 class CifarWorkload(BaseCifarWorkload):
 
+  def __init__(self):
+    super().__init__()
+    self._param_shapes = None
+    self._param_types = None
+    self.epoch_metrics = []
+    self._eval_iters = {}
```

Comment:
All of these attributes besides `self.epoch_metrics` are already defined in `BaseCifarWorkload`. Also, I have noticed that the `__init__()` method of the PyTorch version of the workload can be deleted.

Reply:
Thanks for the check -- fixed!